docs/installation/index.md

```diff
@@ -1,155 +1,149 @@
+![installer start](../images/site-generator.png)
 
-```
-bin/console doctrine:database:create
-bin/console doctrine:schema:create
-bin/console doctrine:fixtures:load
-```
+> Since we answered `yes` to the question `demosite`, the subcommand will be run with the option `--demosite`. This will result in a replica of our [demo site](http://demo.bundles.kunstmaan.be).
 
-![bin/console doctrine:schema:create](https://raw.githubusercontent.com/kunstmaan/KunstmaanBundlesCMS/master/docs/images/schemacreate.png)
+## Initialising the database
 
-## Generate Unit and Behat tests (optional)
+When this is done, the installer will start with the creation of the database schema and load all the generated fixtures to fill it. When the fixtures command is executed, it will ask to purge the database. You can just answer `y` and continue.
 
-If you want, you can generate a set of Unit and Behat test features that test that your site is working correctly. It will test logging in to the administration interface, it will create a page and tries to enter every pagepart. Most generators will generate extra features when you add features later on in development.
+![database setup](../images/database-setup.png)
 
-```
-bin/console kuma:generate:admin-tests
-```
+## Assets install
 
-Just accept the default bundle namespace at the prompt.
+When the database has been setup we will run the assets install command. This will output the compiled assets for the admin section of the KunstmaanCMS. If you want add some extra js to the compiled javascript you can do so as stated in the freshly generated `assets/admin/js/admin-bundle-extra.js` file.
 
-![bin/console kuma:generate:admin-tests](https://raw.githubusercontent.com/kunstmaan/KunstmaanBundlesCMS/master/docs/images/behattests.png)
+> Adding extra css/js to the admin section is a default feature in the KunstmaanCMS frontend setup. If you are not using this setup you will need to implement this logic yourself.
 
+## Admin tests generator
 
-## Get all the front-end CMS assets
+Because at the beginning of the installer we said that we would like to install the admin tests, these will be generated as a last step in the installation procedure. These tests are a set of Unit and Behat test features that test that your site is working correctly. It will test logging in to the administration interface, it will create a page and tries to enter every pagepart. Most generators will generate extra features when you add features later on in development.
 
-Now that all your code is generated, let's make sure all front-end assets are available.
+## Install finished, what now?
 
-Execute the following command:
+![install finished](../images/install_finished.png)
 
-```
-bin/console assets:install --symlink
-```
+Now the install is finished you should be able to browse to the admin section of the website (ie: https://127.0.0.1/en/admin) on the environment of your choice, and be greeted with the following screens. 
```

Comment:
Maybe mention `bin/console server:start` here so that the website is available at 127.0.0.1